penpot_rock/rockcraft.yaml

```diff
@@ -33,6 +33,30 @@ parts:
     override-build: |
       craftctl default
       
+      git apply <<'EOF'
```

Comment:
I'm wondering if this should be kept in a patch file instead?

Reply:
Unfortunately, this part has already used a Git repo as the source, so it can't pull additional files.